pkg/apis/kops/validation/instancegroup.go

```diff
@@ -65,6 +65,10 @@ func ValidateInstanceGroup(g *kops.InstanceGroup) field.ErrorList {
 		allErrs = append(allErrs, field.Invalid(field.NewPath("spec", "rootVolumeIops"), g.Spec.RootVolumeIops, "RootVolumeIops must be greater than 0"))
 	}
 
+	if g.Spec.RootVolumeType != nil {
```

Comment:
do we know if this field is only used by AWS? if there are other cloud providers with different allowed values, we might want to only perform this validation if this is an AWS IG.

Reply:
+1 a quick search shows that aliclound and gcp both use the field.

Reply:
Good catch. Ali also uses this as well.